app/contexts/PassageContext.tsx

```diff
@@ -105,16 +104,16 @@ export function PassageProvider({ children }: { children: React.ReactNode }) {
 
   const fallbackLogin = async (identifier: string) => {
```

Comment:
Since we removed 'PassageAuthFallbackResult' from SDKs. Should we rename this function to avoid confusing new users? what do you think?